src/editor/object_option.cpp

```diff
@@ -440,7 +442,7 @@ FileObjectOption::to_string() const
 void
 FileObjectOption::add_to_menu(Menu& menu) const
 {
-  menu.add_file(get_text(), m_value_pointer, m_filter, m_basedir, m_path_relative_to_basedir);
+  menu.add_file(get_text(), m_value_pointer, m_filter, m_basedir, m_path_relative_to_basedir, -1, m_generate_help_text_for_file);
```

Comment:
`m_generate_help_text_for_file` somehow suggests that this is boolean. No idea if I can think of another name, though.